--- tar.orig	2019-05-25 11:07:43.157634705 +0700
+++ tar	2019-05-25 11:07:58.917761904 +0700
@@ -105,7 +105,7 @@
         # or more options separated by ', ' separator string.  We are matching
         # like this then:  ^<spaces>(<separator>?<option>)+<whatever>$
         if [[ "$line" =~ \
-            ^[[:blank:]]{1,10}(((,[[:blank:]])?(--?([\]\[a-zA-Z0-9?=-]+))(,[[:space:]])?)+).*$ ]]; then
+            ^[[:blank:]]{1,10}(((,[[:blank:]])?(--?([\]\[a-zA-Z0-9?-=]+))(,[[:space:]])?)+).*$ ]]; then
 
             line=${BASH_REMATCH[1]}
             str="${line//,/ }"
